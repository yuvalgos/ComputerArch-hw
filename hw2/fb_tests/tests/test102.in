w 0x773e0
r 0x77384
r 0x773e0
r 0x773bc
w 0x77364
r 0x7735c
w 0x77374
w 0x773b0
w 0x77384
r 0x773b0
r 0x773d0
w 0x773c8
w 0x77370
r 0x77388
r 0x77354
w 0x77364
r 0x773e8
r 0x773e8
r 0x773bc
w 0x773f0
r 0x77358
r 0x7735c
r 0x77398
r 0x773d0
w 0x77374
w 0x773b8
w 0x773ac
w 0x773a8
w 0x773e8
w 0x77358
r 0x77350
r 0x77364
r 0x773ac
w 0x773c4
r 0x77358
r 0x773b0
r 0x773ec
w 0x7735c
w 0x773c8
r 0x77368
r 0x773ec
