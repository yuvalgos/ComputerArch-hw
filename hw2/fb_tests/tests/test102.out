L1miss=0.561 L2miss=1.000 AccTimeAvg=128.512
